modulefiles/ufs_wcoss2.intel.lua

```diff
@@ -1,56 +1,25 @@
 help([[
-loads UFS Model prerequisites on Cactus and Dogwood
+Load environment to build UFS on Acorn with Intel compiler
```

Comment:
should this be Cactus and Dogwood?

Reply:
Yes, this was not changed from me copying the acorn lua file to use for wcoss2.